revolv/base/tests/test_views.py

```diff
@@ -1,9 +1,33 @@
 from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
 from django.test import TestCase
 from revolv.base.models import RevolvUserProfile
 from revolv.lib.testing import TestUserMixin, UserTestingMixin
 
 
+class CategorySetterTestCase(TestUserMixin, UserTestingMixin, TestCase):
+    
+    def test_category_setting(self):
+        self.test_profile.make_administrator()
+        response = self.send_test_user_login_request()
+
+        # checks to see that the user has no pre-existing preferences
+        self.assertEqual(len(self.test_profile.preferred_categories.all()), 0)
+        
+        self.client.post(reverse('dashboard_category_setter'), {'1': '', '2': ''}, HTTP_X_REQUESTED_WITH='XMLHttpRequest')
```

Comment:
Instead of hardcoding the ids like this, can you just grab the first two categories (i.e. `Categories.objects.all()[:2]`) and use their `.id` attribute instead? This way, if we every have to delete and reload categories into the database, this test will still pass, even though the ids will be different